### ISSUE TITLE
Using custom data with 1280 by 720 image size; RuntimeError: Sizes of tensors must match except in dimension 1. Expected size 46 but got size 45 for tensor number 1 in the list.

### ISSUE THREAD

**Issue by FahadMahdi** [Megvii-BaseDetection/YOLOX#1549]
Whenever I try to use one dataset where the image size is 1280 * 720, I kept getting the following error

RuntimeError: Sizes of tensors must match except in dimension 1. Expected size 46 but got size 45 for tensor number 1 in the list.

The error generated in YOLOPAFPN

99   f_out0 = torch.cat([f_out0, x1], 1)  # 512->1024/16

As per my knowledge image size should not be a problem as the model automatically resize it in accordance with its input size.

**Comment by Shaw184** [comment]
Yes I am facing the exact same issue could someone check this out?

**Comment by Shaw184** [comment]
@FahadMahdi this issue can be solved by setting image size to (640,640) or (416,416). If you want multiscale training and want to specify the range of resize checkout the [tutorial to change image size](https://github.com/Megvii-BaseDetection/YOLOX/blob/main/docs/manipulate_training_image_size.md)

**Comment by FateScript** [comment]
The input size of image should be multiple of 32 here.
